config/samples/resources/gkehubfeaturemembership/config-management-feature-membership/gkehub_v1beta1_gkehubmembership.yaml

```diff
@@ -19,7 +19,7 @@ metadata:
     cnrm.cloud.google.com/project-id: gkehubfeaturemembership-dep-acm
   name: gkehubfeaturemembership-dep-acm
 spec:
-  location: global
+  location: us-central1-a
```

Comment:
Should it be `us-central1`? AFAIK, GKEHub membership is a regional resource. And in the test (b/360180935#comment2), the location is `us-central1`.

Reply:
Good catch, fixed that